.github/docker-images/base-images/device-client/amazonlinux/Dockerfile

```diff
@@ -11,25 +11,16 @@ RUN yum -y update \
     bzip2 \
     git \
     wget \
-    curl \
+    perl \
```

Comment:
do you know why these library names have changes? perl, zlib-devel?

Reply:
Changes to the OS required us to start installing perl and zlib-devel. And installing curl no longer works which appears to be a bug with amazonlinux.